src/pages/Login/ResetPassword.styles.scss

```diff
@@ -0,0 +1,65 @@
+$dark: #6831DE;
+$light: #E03BFB;
+.error {
+	margin: 0;
+	padding: 0;
+	.container {
+		min-height: 40vh;
+		padding: 10%;
+		background: #fff;
+		.headline {
+			font-size: 1.6em;
+		}
+
+		.group {
+			width: 80%;
+			display: flex;
+			flex-direction: row;
+			justify-content: space-between;
+
+			.link {
+				color: $dark;
+				padding: 10px;
+				border: 1px solid $light;
+				border-radius: .25rem;
+
+				&:hover {
+					color: $light;
+					border-color: $dark;
+				}
+			}
+		}
+	}
+}
+
+.loading {
+	margin: 0;
+	padding: 0;
+	background: linear-gradient(to right, $light, $dark);
+	.container {
+		box-shadow: none;
+		color: #fff;
+		margin: 0;
+	}
+	img {
+		height: 15vh;
+	}
+}
+
+@media (min-width: 360px) and (max-width: 424px) {
+	.error {
+		padding: 5%;
+	}
+}
+
+@media (min-width: 425px) and (max-width: 767px) {
+	.error {
+		padding: 10%;
+		.container {
+			font-size: 1.3em;
+			.sub-headline {
+				font-size: 0.9em;
+			}
+		}
+	}
+}
```

Comment:
Code not formatted, try running `npm run format`

Reply:
Done!